src/pretix/static/pretixpresale/js/ui/cart.js

```diff
@@ -56,7 +56,7 @@ var cart = {
         } else {
             if (diff_minutes !== cart._prev_diff_minutes) {
                 if (diff_minutes == 0) {
-                    $("#cart-deadline").text(gettext("Your cart is about to expire. If you want to continue, please click here:"))
+                    $("#cart-deadline").text(gettext("Your cart is about to expire.") + " " + gettext("You can renew the reservation period up to ten times."))
```

Comment:
Do we really need to advertise this ten times? Seems irrelevant information to most users and an invitation to some annoying ones.

Reply:
I’d remove the renew-part here completely.
```suggestion
                    $("#cart-deadline").text(gettext("Your cart is about to expire."))
```